r-notebook/Dockerfile

```
@@ -24,9 +24,9 @@ RUN ln -s /bin/tar /bin/gtar
 USER $NB_UID
 
 # R packages
-RUN conda install --quiet --yes \
-    'r-base=3.6.3' \
-    'r-caret=6.0*' \
+RUN conda install --quiet --yes -c conda-forge \
```

It should not be necessary to define the channel since it's the default one. But maybe there is a reason to do it.

```suggestion
RUN conda install --quiet --yes \
```